src/main/java/xyz/nucleoid/bedwars/game/BwMap.java

```diff
@@ -46,13 +48,15 @@
 
     private final LongSet protectedBlocks = new LongOpenHashSet();
 
+    public ItemGeneratorPools pools;
```

Comment:
Very minor nitpick- can we make this field final and be passed in the constructor?